.github/workflows/ci.yml

```diff
@@ -36,9 +36,12 @@ jobs:
           # TODO: Replace with go-version-from-file when it is supported
           # https://github.com/actions/setup-go/pull/62
           go-version: ${{ steps.go-version.outputs.content }}
+      - # since release builds specify the preload tag, we must run the generator
+        name: Generate provider schemas
+        run: go generate ./internal/schemas
       -
         name: Snapshot build (cross-platform)
-        uses: goreleaser/goreleaser-action@v1
+        uses: goreleaser/goreleaser-action@v2
```

Comment:
Good catch! I completely forgot about this other action reference when updating the release job. 😅